pkg/cli/admin/upgrade/status/examples/4.16.0-single-node.output

```diff
@@ -0,0 +1,19 @@
+= Control Plane =
+Assessment:      Completed
+Target Version:  4.16.0-0.nightly-2024-07-24-232111 (a new install)
+Completion:      100%
+Duration:        25m
+Operator Status: 33 Healthy
+
+Control Plane Nodes
+NAME                          ASSESSMENT   PHASE     VERSION                              EST   MESSAGE
+ip-10-0-31-245.ec2.internal   Completed    Updated   4.16.0-0.nightly-2024-07-24-232111   -     
+
+= Worker Upgrade =
+
+WORKER POOL   ASSESSMENT   COMPLETION   STATUS
+worker        Empty                     0 Total
+
+= Update Health =
+SINCE    LEVEL   IMPACT   MESSAGE
+26m19s   Info    None     Update is proceeding well
```

Comment:
First of all, we have discovered an issue in our current code!

The output shows that everything is updated. However, in such a case, it should display the `The cluster is not updating.` message (as it does in the https://github.com/openshift/oc/blob/a5be06f989c6c545b98f0d299f5e905bca53fa10/pkg/cli/admin/upgrade/status/examples/not-upgrading.output example) and not output our beautiful status.

The problem is in the following section of the code: https://github.com/openshift/oc/blob/a5be06f989c6c545b98f0d299f5e905bca53fa10/pkg/cli/admin/upgrade/status/status.go#L270-L276

An empty pool has the `Completion` property set to `NaN`, and thus we proceed to show the overall status. I will create a bug for this issue. It could be a nice opportunity to improve the check as well. As of now, we go through all of the `MachineConfigPools`, `MachineConfigs` and `Nodes` before we decide whether to show the status or display the `The cluster is not updating.` message. However, the `MachineConfigPool` has a nice a field `status.updatedMachineCount` and a condition of the type `Updated`. Maybe we can utilize them in such a way to check whether all nodes are indeed on the desired OCP version and save ourselves a few API calls. However, this is outside of the scope of this PR.

---

Could we modify the examples in such a way that they won't produce the `The cluster is not updating.` message after the above-mentioned bug is fixed? For example, by making a node to be updating or having a degraded operator blocking an update. What do you think?

Reply:
Good catch!
I did feel something was wrong as I did not upgrade the cluster when I got the yamls.
But did not dig further as I saw what I expected to see for the task. Shame on me!

Let me see if I can fix both issues:
- The cluster is not updating.
- Modify one input to show the update status (surviving the fix).

Reply:
Done!